client/src/test/scala/com/gu/anghammarad/JsonTest.scala

```diff
@@ -8,25 +8,29 @@ import org.scalatest.{FreeSpec, Matchers}
 class JsonTest extends FreeSpec with Matchers {
   "targetJson" - {
     "produces valid JSON for a Stack" in {
-      targetJson(Stack("stack")) shouldEqual """{"Stack":"stack"}"""
+      targetJson(List(Stack("stack"))) shouldEqual """{"Stack":"stack"}"""
     }
 
     "produces valid JSON for a Stage" in {
-      targetJson(Stage("stage")) shouldEqual """{"Stage":"stage"}"""
+      targetJson(List(Stage("stage"))) shouldEqual """{"Stage":"stage"}"""
     }
 
     "produces valid JSON for an App" in {
-      targetJson(App("app")) shouldEqual """{"App":"app"}"""
+      targetJson(List(App("app"))) shouldEqual """{"App":"app"}"""
     }
 
     "produces valid JSON for an AwsAccount" in {
-      targetJson(AwsAccount("123456789")) shouldEqual """{"AwsAccount":"123456789"}"""
+      targetJson(List(AwsAccount("123456789"))) shouldEqual """{"AwsAccount":"123456789"}"""
+    }
+    "produces valid JSON for all" in {
+      targetJson(List(Stack("stack"), Stage("stage"), App("app"), AwsAccount("123456789"))) shouldEqual
+        """{"Stack":"stack","Stage":"stage","App":"app","AwsAccount":"123456789"}"""
```

Comment:
I was thinking about making the project depend on circe for tests. We could parse the resulting JSON and inspect the keys, rather than check it as a string. This would also allow a test for "this actually parses" which would have value itself.

Reply:
(the issue here is that the test passing is coupled to the order of a data structure that is technically "unordered")

Reply:
(I know this isn't the only place where this happens and happy for that to be a separate change in a bit)